boards/qorvo/decawave_dwm3001cdk/decawave_dwm3001cdk.dts

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2019 Stéphane D'Alu
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52833_qdaa.dtsi>
+#include "decawave_dwm3001cdk-pinctrl.dtsi"
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "Decawave DWM3001CDK";
+	compatible = "decawave,dwm3001";
+
+	chosen {
+		zephyr,console        = &cdc_acm_uart0;
+		zephyr,shell-uart     = &cdc_acm_uart0;
+		zephyr,uart-mcumgr    = &cdc_acm_uart0;
+		zephyr,bt-mon-uart    = &cdc_acm_uart0;
+		zephyr,bt-c2h-uart    = &cdc_acm_uart0;
+		zephyr,sram           = &sram0;
+		zephyr,flash          = &flash0;
+		zephyr,code-partition = &slot0_partition;
+		zephyr,ieee802154     = &ieee802154;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		// led from top of board down
+		// D20: soemthing related to jlink, red
+		// D13: DW3000 tx(red)/rx(green)
+		// green
+		led0: led_0 {
+			gpios = <&gpio0 04 GPIO_ACTIVE_LOW>;
+			label = "D9 LED";
+		};
+		// red
+		led1: led_1 {
+			gpios = <&gpio0 14 GPIO_ACTIVE_LOW>;
+			label = "D12 LED";
+		};
+		// red
+		led2: led_2 {
+			gpios = <&gpio0 22 GPIO_ACTIVE_LOW>;
+			label = "D11 LED";
+		};
+		// blue
+		led3: led_3 {
+			gpios = <&gpio0 05 GPIO_ACTIVE_LOW>;
+			label = "D10 LED";
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		// SW1 is connected to P0.18, whicy by default is nRESET and
+		// will reset the board
+		button1: button_1 {
+			gpios = <&gpio0 18 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "Push button switch 1";
+			zephyr,code = <INPUT_KEY_0>;
+		};
```

Comment:
this can go, it resets the board, it doesn't need a dts entry

Reply:
If you disable the nRESET you can use it as gpio
But I guess if you do that you could do an overlay?

Reply:
Yes, if someone wanted to do that they can create an overlay for it, they would need to remove the UICR line and would also need to clear the UICR when programming for that change to take effect